docs/java-rest/high-level/search/field-caps.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[java-rest-high-field-caps]]
+=== Field Capabilities API
+
+The field capabilities API allows for retrieving the capabilities of fields across multiple indices.
+
+[[java-rest-high-field-caps-request]]
+==== Field Capabilities Request
+
+A `FieldCapabilitiesRequest` contains a list of fields to get capabilities for,
+should be returned, plus an optional list of target indices. If no indices
+are provided, the request will be executed on all indices.
+
+Note that fields parameter supports wildcard notation. For example, providing `text_*`
+will cause all fields that match the expression to be returned.
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests}/SearchDocumentationIT.java[field-caps-request]
+--------------------------------------------------
+
```

Comment:
shall we add the usual "Optional arguments" section here that describes the indices options arguments?